dataland-frontend/src/components/general/TheSearch.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
The search is currently disabled on the landing page. Therefore, let's move this component here to a different feature branch which holds all the code for the next landing-page-ticket.

Reply:
Yes, we could create a different feature branch with all the components not included in this one.

Reply:
👍   Keeping this open as reminder, until it has been moved